zulipterminal/config/keys.py

```diff
@@ -29,6 +29,11 @@ class KeyBinding(TypedDict):
     # Key that is displayed in the UI is determined by the method
     # primary_key_for_command. (Currently the first key in the list)
 
+    'ACTIVATE_BUTTON': {
+        'keys': ['enter'],
+        'help_text': 'Trigger the selected entry',
+        'key_category': 'general',
+    },
```

Comment:
Re. UI Navigation category,
I can't particularly recall discussing "UI Navigation".
Do you mean moving the narrowing commands in #1516?
We haven't used the term "UI Navigation" there yet, just extracting the narrowing commands from it. I'll update the category name.
I think we could do them independently, and in any order? As it would make sense to have ACTIVATE_BUTTON in navigation category even before it's split.
Please let me know if you have more in mind, as I don't think we've discussed this before.

> space works on buttons

I totally missed that one.
Thank you for that clear guidance. That really helped.
I've added the space key, updated the display function.
I have tied it to the Urwid mapping even though we do not add any new hotkeys at the moment, for consistency and safety.

> document the behavior of enter in the user list to reply/compose to a user. That's distinct from this split, and likely depends on a user-list/panel category.

Yes, I've added that as part of [this commit](https://github.com/zulip/zulip-terminal/pull/1515/commits/625344fa29b22474f0a221b3c41cfa2ade01dabd) in #1515.

Reply:
Re. UI Navigation category,
I can't particularly recall discussing "UI Navigation".
Do you mean moving the narrowing commands in #1516?
We haven't used the term "UI Navigation" there yet, just extracting the narrowing commands from it. I'll update the category name.
I think we could do them independently, and in any order? As it would make sense to have ACTIVATE_BUTTON in navigation category even before it's split.
Please let me know if you have more in mind, as I don't think we've discussed this before.

> space works on buttons

I totally missed that one.
Thank you for that clear guidance. That really helped.
I've added the space key, updated the display function.
I have tied it to the Urwid mapping even though we do not add any new hotkeys at the moment, for consistency and safety.

> document the behavior of enter in the user list to reply/compose to a user. That's distinct from this split, and likely depends on a user-list/panel category.

Yes, I've added that as part of [this commit](https://github.com/zulip/zulip-terminal/pull/1515/commits/625344fa29b22474f0a221b3c41cfa2ade01dabd) in #1515.